File: Beginner Programming Session/temp/task-5.c
// Write a C program that takes two numbers (a, b) as input and then prints the smallest number

#include <stdio.h>

int main() {
    int a, b;
    printf("Enter two numbers: ");
    scanf("%d %d", &a, &b);
    
    printf("%d\n", a < b ? a : b);
    return 0;
}

File: CSE-202 (Object Oriented Programming Lab)/Lab-5/Task5.java
import java.util.*;

// Written by: Md. Maruf Sarker
// Platform: Linux Mint
// Problem: Write a Java program that takes a string as input and tries to parse it into an integer. Handle the number format exception and display an appropriate error message.
// Language: Java
// Date: 2023-05-19
// Difficulty Rating: 

public class Task5 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter a string: ");
        String s = sc.nextLine();

        try {
            int n = Integer.parseInt(s);
            System.out.println("Integer: " + n);
        } catch (NumberFormatException e) {
            System.out.println("Invalid input");
        }

        sc.close();
    }
}